spec/ar_query_matchers/mock_data_model.rb

```diff
@@ -3,6 +3,7 @@
 # @mission Infrastructure
 # @team DEx
 
+require 'logger'
```

Comment:
Do you still need this?

Reply:
Yes, unfortunately there's a requirement from active_record in pre-Rails 7.1 versions :/